en/sensor/airspeed.md

```diff
@@ -43,6 +43,14 @@ Enable each type using its [corresponding parameter](../advanced_config/paramete
 
 Airspeed calibration information can be found in: [Basic Configuration > Airspeed](../config/airspeed.md).
 
+### Parameters
+
```

Comment:
Thanks @avcuenes 

I don't _think_ so. Listing them in this doc means that they should apply to all airspeed sensors. But if you look at the definitions they appear to apply to a particular model the SDP3x, though CAL_AIR_TUBED_MM might be used elsewhere:

- CAL_AIR_CMODEL (INT32)	Airspeed sensor compensation model for the SDP3x
- CAL_AIR_TUBELEN (FLOAT) | Airspeed sensor tube lengthComment: See the CAL_AIR_CMODEL explanation on how 
- CAL_AIR_TUBED_MM (FLOAT) | Airspeed sensor tube diameter. Only used for the Tube Pressure Drop Compensation

So my take is that we should include the ones above that must be set for _all_ airspeed sensors in this page. We might also say that there are other params specific to airspeed setup. 

If any of these must be set for a specific airspeed sensor then they should be mentioned in that sensor.

@sfuhrer Can you please comment on this?

Reply:
I'm pretty sure all 3 are only used for SDP3x sensors.